.appveyor.yml

```diff
@@ -1,3 +1,6 @@
+# Copyright (c) 2020 National Instruments
+# SPDX-License-Identifier: MIT
```

Comment:
Very nice to include the SPDX header 🥇

Reply:
The brevity and ability to automate detection are definitely nice :)